ui/app/templates/components/job-dispatch.hbs

```diff
@@ -0,0 +1,60 @@
+{{#if this.dispatchError}}
+  <div data-test-dispatch-error class="notification is-danger">
+    <h3 class="title is-4" data-test-parse-error-title>Dispatch Error</h3>
+    <p data-test-dispatch-error-message>{{this.dispatchError}}</p>
+  </div>
+{{/if}}
+
+<form action="#" onsubmit="return false">
+  <h1 class="title">Dispatch an instance of '{{ this.model.job.name }}'</h1>
+  <div class="rows">
+    {{#each this.params as |meta|}}
+      <div class="columns">
+        <div class="column is-4">
+          <h2 class="title is-4">{{meta.title}} {{#if meta.isRequired}}*{{/if}}</h2>
+          <input
+            id="{{meta.name}}"
+            style="width: 100%;"
```

Comment:
These `input` fields are not styled like other fields. We normally just use `class="input"` and avoid in-line styles. 

Here's an existing field for reference: https://github.com/hashicorp/nomad/blob/main/ui/app/templates/settings/tokens.hbs#L26